counsel.el

```diff
@@ -1265,7 +1265,7 @@ files in a project.")
   "Default defun to calculate `counsel--git-grep-count'."
   (if (eq system-type 'windows-nt)
       0
-    (read (shell-command-to-string "du -s .git 2>/dev/null"))))
+    (read (shell-command-to-string "du -s `git rev-parse --git-dir` 2>/dev/null"))))
```

Comment:
`$()` syntax is preferred over backticks; see https://unix.stackexchange.com/a/126928/230197.

Reply:
You should also wrap the command substitution in double quotes to handle filenames with spaces in them.

Reply:
Another flexible way of calling `git` from Emacs is via `vc-git-command`.

Pros:
* Allows users to specify their preferred `vc-git-program`.
* Calls `vc-git-program` directly.
* Allows flexible error ignoring/handling via the `OKSTATUS` argument:

      if OKSTATUS is nil, that means to ignore error status, if it is
      `async', that means not to wait for termination of the
      subprocess; if it is t it means to ignore all execution errors

Cons:
* Requires ex- or implicitly loading `vc-git`.
* No other part of Counsel uses `vc`/`vc-git`.

Example:

```el
(with-temp-buffer
  (vc-git-command t t () "rev-parse" "--git-dir")
  (buffer-substring (point-min) (1- (point-max))))
```